src/mmw/js/src/compare/views.js

```diff
@@ -472,8 +534,74 @@ function getTr55Tabs(scenarios) {
                 })
             },
         ],
-        // TODO Make Runoff charts
-        runoffCharts = [],
+        runoffCharts = [
```

Comment:
@rajadain Was there a reason for all the arrays/objects in this function to not be actual models?
 Why doesn't this return a`TabsCollection`?